---
post_url: https://discourse.onlinedegree.iitm.ac.in/t/tds-official-project1-discrepencies/171141/358
reply_to_post_number: 355
---
Hi Hari,

Your docker failed to build.

Did you try to replicate the test environment as mentioned in


[Tds-official-Project1-discrepencies](https://discourse.onlinedegree.iitm.ac.in/t/tds-official-project1-discrepencies/171141/316) [Tools in Data Science](/c/courses/tds-kb/34)

> To replicate the test environment:
> Fetch the github repo’s latest commit before 18th feb use below code for that
> import requests
> import pandas
> DEADLINE = pd.Timestamp("2025-02-18", tz="Asia/Kolkata")
> url = f"https://api.github.com/repos/{owner}/{repo}/commits"
> try :
> response = requests.get(url,headers=github\_headers, timeout=60)
> fetch\_commit = None
> if response.status\_code == 200:
> commits = response.json()
> for commit in commits:
> sha = commit["sha"]
> …

If you tried you would find that it will not build. Thats why you have no logs.  
90 such cases are there where the image could not be built from your repo.

The specific error in your case is:  
tried copying requirements.txt which doesn’t exists

Thats why there are no logs.  
Kind regards